develop/devguide/Automation/UIBlockTypesOverview.md

```diff
@@ -270,6 +270,11 @@ All files uploaded by users will by default be placed in the *C:\\Skyline DataMi
 > [!TIP]
 > See also: [GetUploadedFilePath](xref:Skyline.DataMiner.Automation.UIResults#Skyline_DataMiner_Automation_UIResults_GetUploadedFilePath_System_String_)
 
+> [!NOTE]
+> Contrary to the other UI block types the FileSelector doesn't allow to set an initial value. During an automation script session, it is however possible (since RN37620) to keep the files that were already uploaded after the UI was shown.
```

Comment:
it is however possible (since RN37620) -> Starting from DataMiner xxx, it is possible to...